sopel/plugins/__init__.py

```diff
@@ -80,6 +83,18 @@ def find_sopel_modules_plugins():
             yield handlers.PyModulePlugin(name, 'sopel_modules')
 
 
+def find_entrypoints_plugins(group='sopel.plugins'):
+    """List plugins from a setuptools entry point group
+
+    :param str group: setuptools entry point group to look for
+                      (default to ``sopel.plugins``)
+    :return: Yield instance of :class:`~.handlers.EntrypointPluginHandler`
+             created from setuptools entry point given ``group``
+    """
+    for entrypoint in pkg_resources.iter_entry_points(group):
```

Comment:
```suggestion
    for entry_point in pkg_resources.iter_entry_points(group):
```

And the reference to it on the next line, of course.